src/plus/gk/models/organization.ts

```diff
@@ -29,7 +29,18 @@ export enum OrganizationType {
 
 export type OrganizationConnection = Record<string, unknown>;
 
+export type OrganizationMemberStatus = 'activated' | 'pending';
+
 export interface OrganizationMember {
+	readonly id: string;
+	readonly email: string;
+	readonly name: string;
+	readonly username: string;
+	readonly role: OrganizationRole;
+	readonly status: OrganizationMemberStatus;
+}
+
+export interface OrganizationMemberOld {
```

Comment:
If we're not using this anywhere else, maybe we just chuck it? Or at least comment that it's deprecated and should be removed soon?

Reply:
Missed it, thanks.